addons/mail/static/src/core/common/store_service.js

```diff
@@ -49,6 +49,17 @@ export class Store extends BaseStore {
         return this.imStatusTrackedPersonas.map((persona) => persona.id);
     }
 
+    allSelfChannels = Record.many("Thread", {
+        inverse: "storeAsAllSelfChannels",
```

Comment:
Do we actually need the inverse? `onAdd`/`onDelete` could be added on the other relation, or is it still bugged for `one`?

Reply:
Boolean used, works fine